course-definition.yml

```diff
@@ -751,3 +758,166 @@ stages:
 
     marketing_md: |-
       In this stage, you'll implement support for executing a quoted executable.
+
+  - slug: "jv1"
+    primary_extension_slug: "redirection"
+    name: "Redirect stdout"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for redirecting the output of a command to a file.
+      The `1>` operator is used to redirect the output of a command to a file.
+      But, as a special case, if the file descriptor is not specified before the operator `>`, the output is redirected to the standard output by default, so `>` is equivalent to `1>`.
+      You can read more about Redirecting output [here](https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Output).
```

Comment:
> Redirecting output

Perhaps all lowercase within the sentence could look better?

Reply:
Was the intention to make the title stand out a bit more? How about this:

```
Learn more about [Redirecting Output](https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Output).
```

Reply:
Yep, this sounds better!
Updated.